tests/sanity/ignore-2.10.txt

```diff
@@ -150,6 +150,8 @@ plugins/module_utils/univention_umc.py future-import-boilerplate
 plugins/module_utils/univention_umc.py metaclass-boilerplate
 plugins/module_utils/vexata.py future-import-boilerplate
 plugins/module_utils/vexata.py metaclass-boilerplate
+plugins/action/iptables_state.py action-plugin-docs
```

Comment:
This entry can stay for now, since this is due to a bug of ansible-test. The other one has to go, though.